.github/workflows/build-on-windows.yml

```diff
@@ -0,0 +1,29 @@
+name: Build gProfiler on Windows
+
+on: pull_request
+# on:
+#   pull_request:
+#   push:
+#     tags:
+#       - 'windows'
+
+jobs:
+  build-executable:
+   runs-on: windows-latest
+   steps:
+      - name: Checkout Code
+        uses: actions/checkout@v3
+        with:
+          submodules: true
+
+      - name: install chocolatey
```

Comment:
Shouldn't be necessary to install `chocolatey`, try removing.
According to documentation on Github `choco` [should already be present on workers running windows-latest](https://github.com/actions/runner-images/blob/main/images/win/Windows2022-Readme.md#package-management).